### ISSUE TITLE
LocMemStorage is not used for different model classes heriting the same base model

### ISSUE THREAD

**Issue by xnopre** [danifus/django_override_storage#8]
I have a base model `class MyBase(models.Model)` containing a FileField `file = models.FileField(...)`. I have 3 model classes heriting from `MyBase` like this :
```
class Model1(MyBase):
    pass

class Model2(MyBase):
    pass

class Model3(MyBase):
    pass
```

In my unit tests, when a create and save files, only those for `Model1` are created in memory, files for the 2 other model classes are created in my disk. 

Adding some prints in `def filefields(self):`, I can see that `Model1` is the first processed, and I can see that there are some `if` in `def setup_storage(self)` or `def set_storage(self, field):`, and perhaps there is something working wrong in link with the `file` attribute "shared" between the 3 models classes heriting the same base class ? 

Thanks for your help and perhaps a fix for this bug 😀

**Comment by danifus** [comment]
Thanks for the bug report! I've added a couple of tests demonstrating the bug and will release a fix soon

**Comment by danifus** [comment]
Fixed in 8c046a0fe4876953539da31f9ef81ec535e5cb9b
